pychromecast/controllers/media.py

```diff
@@ -67,6 +69,16 @@ def __init__(self):
         self.media_metadata = {}
         self.subtitle_tracks = {}
         self.current_subtitle_tracks = []
+        self.last_updated = None
+
+    def get_seek_time(self):
```

Comment:
I am not sure about the name 'seek_time'. Also, why would we make it a method and not just a property like the other ones.

I would expect it to be:
 - called `adjusted_current_time`. Make sure to add in the comment why it is called adjusted (it calculates current time)
 - be a property

Reply:
I had it as a property originally, but it wasn't working (likely just beginner error, still figuring things out). I''ll look into the differences between properties and methods in python. Thanks for all the advice!